packages/utilities/psammead-styles/index.test.jsx

```diff
@@ -5,16 +5,28 @@ import * as fonts from './fonts';
 import * as fontsFromSrc from './src/fonts';
 
 const fontsExpectedExports = {
-  FF_NEWS_SANS_LIT: 'string',
-  FF_NEWS_SANS_REG: 'string',
-  FF_NEWS_SANS_MDM: 'string',
-  FF_NEWS_SANS_BLD: 'string',
-  FF_NEWS_SERIF_LIT: 'string',
-  FF_NEWS_SERIF_REG: 'string',
-  FF_NEWS_SERIF_MDM: 'string',
-  FF_NEWS_SERIF_BLD: 'string',
-  FF_NEWS_SANS_REG_ITAL: 'string',
-  FF_NEWS_SERIF_MDM_ITAL: 'string',
+  T_REITH_SERIF_REGULAR: 'string',
```

Comment:
These should either remain `FF_` namespaced or become `F_` namespaced. Prefixing these Typography doesn't make much sense IMO  
FF = Font-Face
F = Font
T = Typography